### DISCUSSION TITLE
Samsung Internet works with DOM Overlays but why not available?

### DISCUSSION THREAD

**Discussion by uprealdigital** [google/model-viewer#2939] [Q&A] [answered]
Hi!

I tested the DOM Overlays with Samsung Internet with the first Augmented Reality Example. Even though it says that the DOM Overlay is Not Available, I do see the different 3D models, the "need scanning hand+phone" image when it's not tracking. Are those not DOM Overlays? Am I missing something here? I tested on my Samsung S7 Edge phone.

![image](https://user-images.githubusercontent.com/25235922/139557608-4b70f1e9-cf76-44b9-8ce5-de28867adb0c.png)

![image](https://user-images.githubusercontent.com/25235922/139557622-c13774e3-65d4-4824-a2aa-cefa9701102d.png)

**Comment by elalish** [comment, marked as answer]
I just haven't updated that list in a year or so, thanks for the ping. Indeed, sounds like it's fully supported now.

**Reply by uprealdigital** [reply]
Ah ok! That's great!
